### ISSUE TITLE
some stuff that i'm working on that blocky might possibly be interested in

### ISSUE THREAD

**Issue by fluxionary** [obl3pplifp/BlS-issue-tracker#379]
* [ ] [moreblocks (/stairsplus) redo](https://github.com/minetest-mods/moreblocks/pull/191), which is still un-merged, but is likely to be
* [x] [my smartshop fork](https://github.com/fluxionary/minetest-smartshop) got a lot of new updates, including one to further reduce FPS lag due to shop entities
* [x] [mesecons_debug](https://github.com/minetest-monitoring/mesecons_debug), which is more a tool for throttling mesecons than debugging anything
* [ ] [mobs_balrog](https://github.com/fluxionary/mobs_balrog), in case you wanted something nasty to put in the nether

**Comment by oversword** [comment]
@fluxionary RE: moreblocks

> adopt current game features which didn't exist when moreblocks was first made

What's the secret sauce?

Others I actually like, except maybe the mesecons one... I'd need to be convinced that was a problem on the server first

I will be (trying) to focus on our nether update in the next few weeks, a balrog might be nice, I'm not sure the currently available mobs I saw were great

**Comment by fluxionary** [comment]
> adopt current game features which didn't exist when moreblocks was first made

things like world-aligned textures. nothing huge, but such small QOL features

**Comment by fluxionary** [comment]
> Others I actually like, except maybe the mesecons one... I'd need to be convinced that was a problem on the server first

you're right, there's probably not much use for it on blocky, at least in the state that i left it - with ABM range severely limited. 

i plan on revisting the issue to create a *generalized* throttling and scheduling mod at some point in the future, but that's at least a year out, probably.
